client/js/components/map/publicdetail/DpPublicDetail.vue

```diff
@@ -68,7 +68,7 @@ export default {
   },
 
   computed: {
-    ...mapState('publicStatement', [
+    ...mapState('PublicStatement', [
```

Comment:
When navigating to public detail in the browser, I get a lot of errors in the console

Reply:
because I just batch-renamed...
Done here Florian Salis core: Adjust naming: SegmentFilter 31 05 2024 14:21

Reply:
still? At least, what I checked, was without errors